src/style.css

```diff
@@ -0,0 +1,145 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+body {
+  font-family: Anenir-Book, sans-serif;
+  font-weight: 300px;
+  font-size: 16px;
+  height: 100vh;
+  background-color: #fff;
+  /* padding: 71px 81px 71px 48px; */
+  margin: 0 auto;
+}
```

Comment:
all of these are redundant, inputs don't inherit font styles other ones are just default
```suggestion
```